test/index.js

```diff
@@ -0,0 +1,54 @@
+const { exec } = require('child_process');
+// eslint-disable-next-line node/no-unpublished-require
+const { test } = require('tapzero');
+const package = require('../package.json');
+
+const { devDependencies, version } = package;
+const FAKE = 'FAKE';
+
+test('should not have dependencies from now until forever', async (t) => {
+  const depKey = 'dependencies';
+  const hasDeps = Object.prototype.hasOwnProperty.call(package, depKey);
+  t.equal(hasDeps, false);
+});
+
+test('should only have ten devDependency from now until forever', async (t) => {
+  const { length } = Object.keys(devDependencies);
+  t.equal(length, 10);
+});
+
+test('should not error when performing a dry-run publish', async (t) => {
+  await new Promise((resolve, reject) => {
+    exec('./scripts/publish.sh', (error) => {
```

Comment:
What if this is `main.sh`, since that is the script that the action will run? Same for the rest of these.

Reply:
updated to test `main.sh` instead